codegen/test-env/server/config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "restApiRoot": "/api",
+  "host": "0.0.0.0",
+  "port": 3000,
```

Comment:
I guess I should have updated this to change the port number (instead of doing it in server.js @Line 12)...